scripts/abci-gen.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This file was added during development of ABCI++. This file is a script to allow
+# the intermediate proto files to be built while active development proceeds
+# on ABCI++.
+# This file should be removed when work on ABCI++ is complete.
+# For more information, see https://github.com/tendermint/tendermint/issues/8066.
+set -euo pipefail
+
+cp ./proto/tendermint/abci/types.proto.intermediate ./proto/tendermint/abci/types.proto
+cp ./proto/tendermint/types/types.proto.intermediate ./proto/tendermint/types/types.proto
+
+MODNAME="$(go list -m)"
+find ./proto/tendermint -name '*.proto' -not -path "./proto/tendermint/abci/types.proto" \
+	-exec sh ./scripts/protopackage.sh {} "$MODNAME" ';'
+
+sh ./scripts/protopackage.sh ./proto/tendermint/abci/types.proto $MODNAME "abci/types"
+
+make proto-gen
+
+mv ./proto/tendermint/abci/types.pb.go ./abci/types
+
+echo "proto files have been compiled"
+
+echo "checking out copied files"
+
+find proto/tendermint/ -name '*.proto' -not -path "*.intermediate"\
```

Comment:
Won't this clobber the output of the code generator?

Reply:
Er, sorry, I meant this comment to be on line 30.

Reply:
No, we copy the file we care about into `abci/types`. For this project, for the time being, we only care about that one file. I'm trying to avoid doing a yak shave of 'regenerate a completely buildable version of all the protos' and just focus on ABCI++ for the moment.

Reply:
Ah, I see. So this is "generate everything and then throw away the stuff that isn't for this context."  Makes sense. 

👍🏼